contrib/podmanimage/README.md

```diff
@@ -1,88 +1,2 @@
-    (especially) security upgrades.  For build details, please [see the
-    configuration file](stable/Containerfile).
-  * `quay.io/containers/podman:latest` and `quay.io/podman/stable:latest` -
-    Built daily using the same Containerfile as above.  The Podman version
-    will remain the "latest" available in Fedora, however the other image
-    contents may vary compared to the version-tagged images.
-  * `quay.io/podman/testing:latest` - This image is built daily, using the
-    latest version of Podman that was in the Fedora `updates-testing` repository.
-    The image is Built with [the testing Containerfile](testing/Containerfile).
-  * `quay.io/podman/upstream:latest` - This image is built daily using the latest
-    code found in this GitHub repository.  Due to the image changing frequently,
-    it's not guaranteed to be stable or even executable.  The image is built with
-    [the upstream Containerfile](upstream/Containerfile). Note the actual compilation
-    of upstream podman [occurs continuously in
-    COPR](https://copr.fedorainfracloud.org/coprs/rhcontainerbot/podman-next/).
-
-## Sample Usage
-
-
-```
-podman pull docker://quay.io/podman/stable:latest
-
-podman run --privileged stable podman version
-
-# Create a directory on the host to mount the container's
-# /var/lib/container directory to so containers can be
-# run within the container.
-mkdir /var/lib/mycontainer
-
-# Run the image detached using the host's network in a container name
-# podmanctr, turn off label and seccomp confinement in the container
-# and then do a little shell hackery to keep the container up and running.
-podman run --detach --name=podmanctr --net=host --security-opt label=disable --security-opt seccomp=unconfined --device /dev/fuse:rw -v /var/lib/mycontainer:/var/lib/containers:Z --privileged  stable sh -c 'while true ;do sleep 100000 ; done'
-
-podman exec -it  podmanctr /bin/sh
-
-# Now inside of the container
-
-podman pull alpine
-
-podman images
-
-exit
-```
-
-**Note:** If you encounter a `fuse: device not found` error when running the container image, it is likely that
-the fuse kernel module has not been loaded on your host system.  Use the command `modprobe fuse` to load the
-module and then run the container image.  To enable this automatically at boot time, you can add a configuration
-file to `/etc/modules.load.d`.  See `man modules-load.d` for more details.
-
-### Blog Post with Details
-
-Dan Walsh wrote a blog post on the [Enable Sysadmin](https://www.redhat.com/sysadmin/) site titled [How to use Podman inside of a container](https://www.redhat.com/sysadmin/podman-inside-container).  In it, he details how to use these images as a rootful and as a rootless user.  Please refer to this blog for more detailed information.
+The podman container image build context and automation have been
+moved to [https://github.com/containers/image_build](https://github.com/containers/image_build)
```

Comment:
LGTM but it might've been friendlier to link directly into the `/podman` subdirectory

Reply:
That's not a bad idea, I can do that.